content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -1,201 +1,209 @@
+By default, the KFP standalone deployment installs an [inverting proxy agent](https://github.com/google/inverting-proxy) that exposes a public URL. If you want to skip the installation of the inverting proxy agent, complete the following:
 
-1. Comment out the proxy component in the [kustomization.yaml](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/kustomization.yaml).
-1. Then run:
+1. Comment out the proxy components in the base [kustomization.yaml](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/kustomization.yaml).
 
-```
-kubectl apply -k manifests/kustomize/env/dev
-# Or the following if using GCP Cloud SQL + Google Cloud Storage
-# kubectl apply -k manifests/kustomize/env/gcp
-```
+1. Apply the changes to update the Kubeflow Pipelines deployment:
 
-The UI is still accessible by port-forwarding:
+     ```
+     kubectl apply -k manifests/kustomize/env/dev
+     ```
 
-```
-kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
-```
+     **Note**: If using GCP Cloud SQL and Google Cloud Storage, apply with this command:
+
+     ```
+     kubectl apply -k manifests/kustomize/env/gcp
+     ```
 
-and open http://localhost:8080/.
+1. Verify that the Kubeflow Pipelines UI is accessible by port-forwarding:
 
-## Uninstall
+     ```
+     kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
+     ```
 
-You can uninstall Kubeflow Pipelines by:
+1. Open the Kubeflow Pipelines UI at `http://localhost:8080/`.
 
-1. Configure kubectl to talk to your cluster. Refer to [Configuring cluster access for kubectl](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl).
+## Uninstalling Kubeflow Pipelines
 
-1. Uninstall Pipelines:
-    ```
-    export PIPELINE_VERSION={{% pipelines/latest-version %}}
-    kubectl delete -k github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
-    ```
+To uninstall Kubeflow Pipelines, run `kubectl delete -k <manifest-file>`.
 
-    Or if you deployed through kustomize:
+For example, to uninstall KFP using manifests from a GitHub repository, run:
 
-    ```
-    kubectl delete -k manifests/kustomize/env/dev
-    # Or the following if using GCP Cloud SQL + Google Cloud Storage
-    # kubectl delete -k manifests/kustomize/env/gcp
-    ```
+```
+export PIPELINE_VERSION={{% pipelines/latest-version %}}
+kubectl delete -k github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+```
+
+To uninstall KFP using manifests from your local repository or file system, run:
+
+```
+kubectl delete -k manifests/kustomize/env/dev
+```
+
+**Note**: If you are using GCP Cloud SQL and Google Cloud Storage, run:
+
+```
+kubectl delete -k manifests/kustomize/env/gcp
+```
+
+## Best practices for maintaining manifests
 
-## Best practices maintaining custom manifests
+Similar to source code, configuration files belong in source control.
+A repository manages the changes to your
```

Comment:
I wonder if these sentences fit with the rest of the doc.

Reply:
Agreed, this section doesn't seem useful. I would recommend that we split this into a separate issue. It seems like this requires guidance on how to customize a kustomize YAML file, then deploy/update/remove with the custom YAML.

What do you think?

Reply:
Right, I don't think this section (Best practices) adds any new information.
The `Customizing` section outlines (could provide more details) several ways to modify manifest files, but does not explain why you should do this. Could provide some examples of customizing manifest files based on pulling in configuration files from other repositories. Customization (and kustomize) is a larger topic. Also, `kustomize` overlays is mentioned, but not in detail.
Do you want the `Best Practices` removed 😃 ?

Reply:
I think that section could use some work. I'll write this up as a separate issue so we can merge these changes.